ebuild-writing/common-mistakes/text.xml

```diff
@@ -380,6 +380,19 @@ Make sure when you bump a version, the stable KEYWORDS are all marked as
 
 </body>
 </subsection>
+<subsection>
+<title>Unused flags in IUSE</title>
+<body>
+
+<p>
+Sometimes vestigal USE flags remain in IUSE despite having no function. Remember
```

Comment:
"vestigal"? That's an uncommon word, so non-native speakers may have to look it up (I had). Also, looks like it's uncommon enough for you to misspell it :smile: (my dictionary says "vestigial").

Reply:
Heh. Okay, good point. Fixing.